app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,10 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-feature
+        android:name="android.hardware.camera"
+        android:required="false" />
```

Comment:
what is the purpose of this?

Reply:
![image](https://github.com/Nub-en-Nuber/NuberJam-Android/assets/49654730/52206260-a3c7-4011-9a1e-7dba882a8eb3)
It's say required

Reply:
okay